### PR TITLE
Fix odd wording

### PR DESCRIPTION
**Pull request by frigginglorious**
Thought the first sentence looked awkward, wanted to see how this worked. Very meta.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # All Contributors v1.0.0-beta.0
 
-This is a specification for recognizing contributors to an open source project in a way that rewards any and every contribution
-whether or not it be code.
+This is a specification for recognizing contributors to an open source project in a way that rewards any and every contribution, not just code.
```
Comment (ben-eb) [inline README.md]: Perhaps *each and every* is more appropriate?

Comment (kentcdodds) [inline README.md]: :+1: I like that better

File: README.md [hunk 2]
```diff
@@ -60,8 +59,8 @@ Emoji | Represents | Links to
...
...its?author=bogas04) 👀 | [![Ben Briggs](https://avatars1.githubusercontent.com/u/1282980?v=3&s=100)<br /><sub>Ben Briggs</sub>](http://beneb.info)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=ben-eb) 👀 | [![James Monger](https://avatars1.githubusercontent.com/u/2037007?v=3&s=100)<br /><sub>James Monger</sub>](http://github.com/Jameskmonger)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=Jameskmonger) | [![Jeroen Engels](https://avatars.githubusercontent.com/u/3869412?v=3&s=100)<br /><sub>Jeroen Engels</sub>](https://github.com/jfmengels)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=jfmengels) [🔧](https://www.npmjs.com/package/all-contributors-cli) 👀 | [![Chris Simpkins](https://avatars0.githubusercontent.com/u/4249591?v=3&s=100)<br /><sub>Chris Simpkins</sub>](http://github.com/chrissimpkins)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=chrissimpkins) 👀 | [![Daniel Kraft](https://avatars1.githubusercontent.com/u/3982200?v=3&s=100)<br /><sub>Daniel Kraft</sub>](http://github.com/frigginglorious)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=frigginglorious) |
```
Comment (kentcdodds) [inline README.md]: :+1:

**Comment by kentcdodds** [comment]
Love this. Could you please update the wording as @ben-eb mentioned and then I say this looks good to me!

File: README.md (hunk 1 above)
Comment (frigginglorious) [inline README.md]: I'd almost agree, but "any" sounds more inclusive. On the other hand, using "each" makes it sound more important. It's a toss-up ;)

**Comment by kentcdodds** [comment]
And by that I mean that once that update has happened, anyone else can feel free to merge this :-)

**Comment by kentcdodds** [comment]
LGTM :+1:

**Comment by jfmengels** [comment]
LGTM :+1: (not a collaborator, so not merging ;) )

**Comment by kentcdodds** [comment]
> not a collaborator, so not merging ;)

Whoops! Fixed that :D

**Comment by kentcdodds** [comment]
> If you have the chance to, do you mind squashing the commits @frigginglorious?

That'd be preferable if you could :-) [learn more](https://egghead.io/lessons/javascript-how-to-squash-multiple-git-commits?series=how-to-contribute-to-an-open-source-project-on-github)

But I'm fine either way

**Comment by kentcdodds** [comment]
@frigginglorious, thanks for trying to clean up the history. Didn't really work out I think, but that's ok :-) I'm fine to merge this. Would you rather that I wait for you to squash your changes into a single commit?

**Comment by Jameskmonger** [comment]
LGTM. I won't merge as @kentcdodds has asked about the commit changes :smile:

Nice work @frigginglorious

**Comment by kentcdodds** [comment]
Sorry! This will now need to be [rebased](https://egghead.io/lessons/javascript-how-to-rebase-a-git-pull-request-branch?series=how-to-contribute-to-an-open-source-project-on-github). I'll just go ahead and fix it, but I'll attribute you as the author! Thanks for the contribution!

**Comment by kentcdodds** [comment]
Closing this in favor of #24.

**Pull request closed without merging**